pallets/runtime/mainnet/src/runtime.rs

```diff
@@ -0,0 +1,378 @@
+    pub const RentDepositOffset: Balance = 300 * DOLLARS;
+    /// Reward that is received by the party whose touch has led
+    /// to removal of a contract.
+    pub const SurchargeReward: Balance = 150 * DOLLARS;
+    */
+
+    // Settlement:
+    pub const MaxLegsInInstruction: u32 = 10;
+
+    // Offences:
+    pub OffencesWeightSoftLimit: Weight = Perbill::from_percent(60) * MaximumBlockWeight::get();
+
+    // I'm online:
+    pub const SessionDuration: BlockNumber = EPOCH_DURATION_IN_SLOTS as _;
+    pub const ImOnlineUnsignedPriority: TransactionPriority = TransactionPriority::max_value();
+
+    // Assets:
+    pub const MaxNumberOfTMExtensionForAsset: u32 = 5;
+    pub const AssetNameMaxLength: u32 = 128;
+    pub const FundingRoundNameMaxLength: u32 = 128;
+
+    // Compliance manager:
+    pub const MaxConditionComplexity: u32 = 50;
+
+    // Corporate Actions:
+    pub const MaxTargetIds: u32 = 1000;
+    pub const MaxDidWhts: u32 = 1000;
+
+    // Statistics:
+    pub const MaxTransferManagersPerAsset: u32 = 3;
+
+    // Scheduler:
+    pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * MaximumBlockWeight::get();
+    pub const MaxScheduledPerBlock: u32 = 50;
+
+    // Identity:
+    pub const InitialPOLYX: Balance = 0;
+
+    /*
+    /// The fraction of the deposit that should be used as rent per block.
+    pub RentFraction: Perbill = Perbill::from_rational_approximation(1u32, 30 * DAYS);
+    // The lazy deletion runs inside on_initialize.
+    pub DeletionWeightLimit: Weight = AVERAGE_ON_INITIALIZE_RATIO *
+        RuntimeBlockWeights::get().max_block;
+    // The weight needed for decoding the queue should be less or equal than a fifth
+    // of the overall weight dedicated to the lazy deletion.
+    pub DeletionQueueDepth: u32 = ((DeletionWeightLimit::get() / (
+                <Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(1) -
+                <Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(0)
+                )) / 5) as u32;
+    */
+}
+
+/// Splits fees 80/20 between treasury and block author.
+pub type DealWithFees = SplitTwoWays<
+    Balance,
+    NegativeImbalance<Runtime>,
+    _4,
+    Treasury, // 4 parts (80%) goes to the treasury.
+    _1,
+    Author<Runtime>, // 1 part (20%) goes to the block author.
+>;
+
+// Staking:
+pallet_staking_reward_curve::build! {
+    const REWARD_CURVE: PiecewiseLinear<'_> = curve!(
+        min_inflation: 0_025_000,
+        max_inflation: 0_140_000,
+        ideal_stake: 0_700_000,
+        falloff: 0_050_000,
+        max_piece_count: 40,
+        test_precision: 0_005_000,
+    );
+}
+parameter_types! {
+    pub const SessionsPerEra: sp_staking::SessionIndex = 6;
+    pub const BondingDuration: pallet_staking::EraIndex = 28;
+    pub const SlashDeferDuration: pallet_staking::EraIndex = 14; // 1/4 the bonding duration.
```

Comment:
```suggestion
    pub const SlashDeferDuration: pallet_staking::EraIndex = 14; // 1/2 the bonding duration.
```

Reply:
This file shouldn't have been pulled into this PR.